### ISSUE TITLE
Lost extended mod card timeout options.

### ISSUE THREAD

**Issue by Dragoonseal** [night/betterttv#4795]
I only have the two generic timeout/ban buttons at the moment. Normally BTTV ads buttons for !permit, purge, 10 minute timeout, 1 hour timeout, 8 hour timeout, 24 hour timeout.

**Comment by night** [comment]
fixed via https://github.com/night/betterttv/commit/c48e36da4eec3bbaa675178b4d3678a875bc43e9

**Comment by Dragoonseal** [comment]
@night This didn't fix the issue. I let it sit for a day in case it was a caching issue, but I've been on Version 7.4.1 for a day now and the buttons are still missing.

**Comment by night** [comment]
If you have steps to reproduce I can take a look, but they are working on the production 7.4.1 release for me.

**Comment by Dragoonseal** [comment]
OS: Win10
Chrome: Version 93.0.4577.63 (Official Build) (64-bit)
BTTV: Version 7.4.1 • Sep 12, 2021 (I was on Version 7.4.0 when the buttons first went away)

A tab popped up the other day asking me to grant BTTV new permissions for Youtube emotes, but I don't use Youtube so I closed the tab. I haven't found any option in BTTV or in Chrome's extension options to update permissions since I closed the tab. 

Not sure if that has bearing on the mod card issue though. But I've not changed any other settings, so I don't know how to reproduce. I just started up Chrome yesterday and the permit/timeout buttons were gone. If there's anything I can do to help reproduce or narrow down the issue, let me know.
